source/security/enterprise-authentication.txt

```diff
@@ -92,6 +92,8 @@ Complete the following steps to authenticate with GSSAPI:
          You must replace the ``@`` symbol in the principal with ``%40``, as shown
          in the preceding example.
 
+.. include:: /includes/security/auth-properties-commas.rst
```

Comment:
The placement of this admonition seems a bit strange. Can/should it be moved into the same section/paragraph as the note above?

![image](https://github.com/user-attachments/assets/9693e3d0-7424-4e85-8dbf-32031312a5c4)

Reply:
Ah yep, it needs to be indented